src/gui/src/drcWidget.cpp

```diff
@@ -308,6 +308,14 @@ void DRCWidget::setLogger(utl::Logger* logger)
 
 void DRCWidget::selectReport()
 {
+  if (!block_) {
+    try {
+      logger_->error(utl::GUI, 104, "No database has been loaded");
+    } catch (const std::runtime_error& e) {
+    }
```

Comment:
This makes no sense.  Drop the try-catch.  As written this is equivalent to a warning.  Why shouldn't it be an exception that throws?

Reply:
I put a try-catch because, contrary to a warn, it triggers an exception and crashes:

```
[ERROR GUI-0104] No database has been loaded
terminate called after throwing an instance of 'std::runtime_error'
  what():  GUI-0104
Signal 6 received
```

This is how errors works no?

Reply:
@maliberty the GUI doesn't naturally handle the exceptions like the TCL interface does. What is there is equivalent to other places in the GUI code where Openroad might throw and exception and it needs to be handled properly.

Reply:
See #5624